packages/stencil/src/components/datacom-link/datacom-link.css

```diff
@@ -0,0 +1,168 @@
+:host {
+  --link-color: var(--dc-primary-colour);
+  --link-visited-color: #6b2fc6;
+  --link-font-size: 18px;
+  --link-height: 24px;
+  --link-icon-hover-color: #0a1840;
+}
+
+.dc-link {
+  font-family: var(--dc-font-family);
+  font-style: normal;
+  font-weight: 500;
+  display: inline-flex;
+  flex-wrap: wrap;
+  text-align: center;
+  position: relative;
+  text-decoration: none;
+  white-space: nowrap;
+  vertical-align: center;
+  color: var(--link-color);
+  font-size: var(--link-font-size);
+  line-height: var(--link-height);
+  letter-spacing: -0.5px;
+}
+
+.dc-link-inline {
+  --link-font-size: 16px;
+  --link-height: 22px;
+
+  border-bottom: 1px solid var(--dc-primary-colour);
+}
+
+.dc-link-stacked,
+.dc-link-footer {
+  --link-color: var(--dc-primary-text-color);
+}
+
+.dc-link-footer {
+  --link-font-size: 14px;
+  --link-height: 18px;
+}
+
+.dc-link-icon path {
+  fill: var(--link-color);
+}
+
+.dc-link-stacked,
+.dc-link-footer .dc-link-icon path {
+  fill: var(--dc-primary-text-color);
+}
+
+.dc-link-icon {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  align-self: center;
+}
+
+.dc-link-footer .dc-link-icon {
+  height: 14px;
+  width: 14px;
+}
+
+.dc-link-image-right {
+  display: inline-flex;
+  flex-direction: row-reverse;
+}
+
+.dc-link-image-left {
+  display: inline-flex;
+}
+
+.dc-link-image-left .dc-link-icon {
+  padding-right: 4px;
+}
+
+.dc-link-image-right .dc-link-icon {
+  padding-left: 4px;
+}
+
+.dc-link:visited {
+  color: var(--link-visited-color);
+}
+
+.dc-link:visited .dc-link-icon path {
+  fill: var(--link-visited-color);
+}
+
+.dc-link-inline:visited {
+  border-bottom: var(--link-visited-color);
```

Comment:
This should just be border-color. (But perhaps this might not matter, considering my previous comment about removing borders entirely.)